pkgs/applications/radio/qdmr/default.nix

```diff
@@ -0,0 +1,64 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  makeDesktopItem,
+  writeText,
+  cmake,
+  wrapQtAppsHook,
+  libusb1,
+  libyamlcpp,
+  libsForQt5,
```

Comment:
since you're using `libsForQt5.callPackage`, you can/should add `qtlocation`, `qtserialport` etc here directly.